mkdocs/docs/code-samples/knative/canary-fixedsplit.md

```diff
@@ -35,7 +35,7 @@ kubectl wait ksvc/sample-app --for condition=Ready --timeout=120s
 kustomize build $ITER8/samples/knative/canaryfixedsplit/experimentalservice | kubectl apply -f -
 ```
 
-??? info "Look inside output of `kustomize build $ITER8/.../baseline`"
+??? info "Look inside the output of the `baseline`"
```

Comment:
Extra attention

Reply:
We want to look inside the output of a command... the command is kustomize build (not baseline).